eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -120,7 +120,11 @@ function Confirm-ChangeLogEntry {
     else {
       $status = $changeLogEntry.ReleaseStatus.Trim().Trim("()")
       try {
-        [DateTime]$status
+        $releaseDate = [DateTime]$status
+        if (((Get-Date).AddDays(-31) -gt $releaseDate) -or ($releaseDate -gt (Get-Date).AddDays(31)))
+        {
+          throw "Date is way outside of the current release period."
```

Comment:
This throw will to caught and ignored in the catch block below. We should just add"
```
LogError "Invalid date [ $status ] passed as for Version [$($changeLogEntry.ReleaseVersion)]. The date must be within +/- one month from today." 
return $false
```